cg/store/crud/read.py

```diff
@@ -1413,14 +1413,17 @@ def get_orders_by_ids(self, order_ids: list[int]) -> list[Order]:
             ids=order_ids,
         ).all()
 
-    def get_order_by_id(self, order_id: int) -> Order | None:
+    def get_order_by_id(self, order_id: int) -> Order:
         """Returns the entry in Order matching the given id."""
         orders: Query = self._get_query(table=Order)
         order_filter_functions: list[Callable] = [OrderFilter.BY_ID]
         orders: Query = apply_order_filters(
             orders=orders, filters=order_filter_functions, id=order_id
         )
-        return orders.first()
+        order: Order | None = orders.first()
+        if not order:
```

Comment:
```suggestion
        if not order := orders.first():
```
Might be nice? If the assignment scope extends past the if-statement?

Reply:
I think it looks less readable, prefer this syntax in this context.
```python
  if not (order := orders.first()):
```
The assignment is not as clear.

Reply:
Changed in https://github.com/Clinical-Genomics/cg/pull/3095/commits/41ede3a4ff2901f1ea877606b8516cf85e1491a1